HVSDRA/hero-vs-dragon/damage_to_hero_armor.go

```diff
@@ -1,14 +1,20 @@
 package main
 
+import "fmt"
+
 func damageToArmor(hero Hero, dragon Dragon) (heroArmor int, heroHealth int) {
 	damageToArmor := dragon.damage
 	if hero.armor-damageToArmor > 0 {
 		hero.armor -= damageToArmor
-	} else {
+	} else if hero.armor-damageToArmor < 0 {
 		damageToArmor = (0 - hero.armor) * (-1)
```

Comment:
Does it equal to just hero.armor? Why use all there calculations? It is really hard to read the code...